securedrop_export/export.py

```diff
@@ -23,6 +25,41 @@
 logger = logging.getLogger(__name__)
 
 
+class ExportStatus(Enum):
+
+    # General errors
+    ERROR_FILE_NOT_FOUND = 'ERROR_FILE_NOT_FOUND'
+    ERROR_EXTRACTION = 'ERROR_EXTRACTION'
+    ERROR_METADATA_PARSING = 'ERROR_METADATA_PARSING'
+    ERROR_ARCHIVE_METADATA = 'ERROR_ARCHIVE_METADATA'
+    ERROR_USB_CONFIGURATION = 'ERROR_USB_CONFIGURATION'
+    ERROR_GENERIC = 'ERROR_GENERIC'
+
+    # USB preflight related errors
```

Comment:
we talked already about this but it'll be a readability improvement, after this change, to move lsblk check into the usb disk preflight check and create a new printer preflight check and to use status codes:
`ERROR_DISK_NOT_FOUND`
`ERROR_DISK_PREFLIGHT_CHECK`
`ERROR_PRINTER_NOT_FOUND`
`ERROR_PRINTER_PREFLIGHT_CHECK`